### PR TITLE
Adding Aspire host project

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Adds Aspire host project. This is a powerful project which will make development much easier since you can run just about any dependency you want like Redis...

image

### CODE REVIEW

**Comment by MikeAlhayek** [comment]
@sebastienros Aspire is setup here and looks amazing.

At some point, we may want to add YARP to the CMS app so that we can run the CMS with 3 replicas and then YARP will handles distributing the requests. This would be helpful to test out app distribution services.

**Comment by MikeAlhayek** [comment]
This PR is moving to https://github.com/dotnet/aspire-samples/pull/77

File: src/OrchardCore.Mvc.Web/Program.cs [hunk 1]
```diff
@@ -15,4 +15,4 @@
 
 app.UseOrchardCore();
 
-app.Run();
+await app.RunAsync();
```
Comment (Piedone) [inline src/OrchardCore.Mvc.Web/Program.cs:18]: These would be useful to keep.

**Comment by Piedone** [comment]
So I guess this should be closed, with perhaps some docs suggesting people to check out the Aspire sample? And the `RunAsync()` changes would be useful.

**Comment by MikeAlhayek** [comment]
Yes. I actually want to keep all the changes beside the Aspire project. I'll just need to fine time to clean it up to keep the MVC changes

**Comment by Piedone** [comment]
Great then! Feel free to request my review then.

**Comment by MikeAlhayek** [comment]
closing this PR. moved anything we would keep into PR #15115

**Pull request closed without merging**